Gruntfile.js

```diff
@@ -601,6 +601,24 @@ module.exports = function(grunt) {
           ].join(' && ');
         }
       },
+      'unit-webapp': {
+        cmd: () => {
+          return [
+            `cd webapp`,
+            '../node_modules/.bin/ng test webapp --watch=false',
+            'cd ../',
+          ].join(' && ');
+        }
+      },
+      'unit-webapp-continuous': {
+        cmd: () => {
+          return [
+            `cd webapp`,
+            '../node_modules/.bin/ng test webapp --watch=true',
+            'cd ../',
+          ].join(' && ');
+        }
+      },
```

Comment:
I feel like there should be a better way, but as long as this runs it'll do for now and we can figure out the "right" way later.

Reply:
Yes, agree, maybe we can create a task for further research. We need the angular cli for running unit test.